lambdas/cli.py

```diff
@@ -393,6 +393,8 @@ def validate_aip_bulk_worker(
         error_msg = f"Error validating AIP {aip_uuid or s3_uri}: {exc}"
         logger.error(error_msg)
         with results_lock:
+            results_df.loc[row_index, "aip_uuid"] = aip_uuid
+            results_df.loc[row_index, "aip_s3_uri"] = s3_uri
```

Comment:
This makes sense to me.

If the function `validate_aip_via_lambda` got updated to include the AIP uuid + S3 URI in the response from [this past PR](lambda-response-exception), it makes sense to capture that output in this bulk handler to write to the output.